README.md

```diff
@@ -7,7 +7,45 @@
 [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/MotionAnimator.svg)](https://cocoapods.org/pods/MotionAnimator)
 [![Platform](https://img.shields.io/cocoapods/p/MotionAnimator.svg)](http://cocoadocs.org/docsets/MotionAnimator)
 
-
+<table>
+  <tr><td>🎉</td><td>Implicit and explicit additive animations.</td></tr>
+  <tr><td>🎉</td><td>Parameterized motion with the <a href="https://github.com/material-motion/motion-interchange-objc">Interchange</a>.</td></tr>
+  <tr><td>🎉</td><td>Provide velocity to animations directly from gesture recognizers.</td></tr>
+  <tr><td>🎉</td><td>Maximize frame rates by relying more on Core Animation.</td></tr>
+  <tr><td>🎉</td><td>Animatable properties are Swift enum types.</td></tr>
+  <tr><td>🎉</td><td>Consistent model layer value expectations.</td></tr>
+</table>
```

Comment:
Nit: I would suggest changing this to a bulleted list. In VoiceOver it reads as "party popper", and the next element is the text cell.

Reply:
Tried a list, table without emoji, and table with emoji, and table with emoji was the only variant that attracted the attention I'm attempting to grab here.

Reply:
Ack.